src/pudl/etl/denormalized_assets.py

```diff
@@ -1,41 +0,0 @@
-"""Denormalized tables, based on other simple normalized tables.
```

Comment:
This module was just a placeholder. Output assets are being added to the DAG in the `pudl.ouput` subpackage.